src/component/elements/Sections.tsx

```diff
@@ -1,3 +1,8 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {
+  draggable,
+  dropTargetForElements,
+} from '@atlaskit/pragmatic-drag-and-drop/element/adapter';
```

Comment:
This import will make the build not native ESM compatible.
@targos I don't know if it really matters now that we are not using Next.js anymore.

See https://github.com/zakodium-oss/react-science/issues/899 and the related issues and resolution PR.

Reply:
This is more important without Next.js

Reply:
We can move to another package that supports ESM, like https://github.com/clauderic/dnd-kit

Reply:
I would not use a different library in react-science and here.

Also dnd-kit has a lot of known performance issues and is currently in a big rewrite that is taking a lot of time to complete. I would not use it right now.

Reply:
I do not think that @atlaskit/pragmatic-drag-and-drop will resolve the issue soon, as a related problem has been open in their repository since [April 7, 2024](https://github.com/atlassian/pragmatic-drag-and-drop/issues/27#issue-2229922572) with no updates or activity

Reply:
Then, I will follow your way, and I hope they will resolve the issue in the coming days.